examples/ode_models/ODE_with_manual_gradients.ipynb

```diff
@@ -6,6 +6,9 @@
    "metadata": {},
```

Comment:
<p>This plot isn't displaying for some reason.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/23/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>Yeah, I realized I didn't properly re-run the notebook. In order to do so, I resolved issues like:</p><ul><li>Specifying <code>core</code>  param on <code>pm.sample</code></li><li>Update submodule ref (e.g From theano.Op to theano.tensor.Op)</li></ul><p> </p>